.ipynb_checkpoints/command_analogues/ls.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+
+class bcolors:
```

Comment:
А вот классы в питоне пишутся с большой буквы)
```suggestion
class Bcolors:
```